components/raftstore/src/store/worker/metrics.rs

```diff
@@ -10,10 +10,11 @@ use tikv_util::time::Instant;
 make_auto_flush_static_metric! {
     pub label_enum SnapType {
        generate,
-       apply,
+       apply
     }
 
     pub label_enum SnapStatus {
+        total,
```

Comment:
what's different between `total` and `all`?

Reply:
total means that the task is not runing, it needs to wait.
all means that the task is runnig . 
the sub of them is the waiting count .

Reply:
they are not very distinguishable. How about change `all` to `running`? I think origin `all` is unreasonable.

Reply:
fix